src/applications/vaos/tests/appointment-list/components/CommunityCareAppointmentDetailsPage.unit.spec.js

```diff
@@ -595,8 +595,6 @@ describe('VAOS Page: CommunityCareAppointmentDetailsPage with VAOS service', ()
     expect(screen.queryByTestId('add-to-calendar-link')).not.to.exist;
   });
 
-  it('should navigate to community care appointments detail page', async () => {});
-
   it('should fire a print request when print button clicked', async () => {});
```

Comment:
I'm a huge anti-fan of placeholder tests that will eventually be added or converted to e2e tests. This is very hard to track/remember. I did a quick look through our board and I did not see anything tracking this work (unless it's being tracked as a subitem of an epic that I missed). 

Instead, we should track the work in an issue with detail on what tests are required, something like https://github.com/department-of-veterans-affairs/va.gov-team/issues/39938 but with more details hopefully. We should also remove these empty tests since they only inflate our test numbers without providing any benefit.

If folks agree, I can do a quick scan of all unit tests and file the appropriate issues.

Reply:
@JunTaoLuo - Looks like 'it should navigate to community care appointments detail page' is a duplicate' and should be deleted and 'it should  fire a print request when print button clicked' should be implemented.

Reply:
I'll probably file followup issues for these placeholders tests in a separate PR since I want to systematically examine all of our test files, hope that's okay.